packages/react-search-ui-views/src/Result.js

```diff
@@ -86,25 +86,8 @@ function Result({
 
       <div className="sui-result__body">
         {thumbnail && (
-          <div
-            className="sui-result__image"
-            style={{
-              maxWidth: "140px",
-              paddingLeft: "24px",
-              paddingTop: "10px"
-            }}
-          >
-            <img
-              src={thumbnail}
-              alt="thumbnail"
-              style={{
-                display: "block",
-                width: "100%",
-                height: "100%",
-                objectFit: "cover",
-                objectPosition: "center"
-              }}
-            />
+          <div className="sui-result__image">
+            <img src={thumbnail} alt="" />
```

Comment:
Quick note from an accessibility point of view @JasonStoltz - we might want to consider providing a "thumbnailDescription" field so that people can inject their own accessible descriptions of thumbnails. However, we definitely don't want something generic like "thumbnail" since that is meaningless to a screen reader user - we either want a useful description, or a blank alt (an empty `""` tells a screen reader there's nothing to read out).